com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/v2/ApplicationMetadataBuilder.java

```diff
@@ -0,0 +1,72 @@
+package com.sap.cloud.lm.sl.cf.core.cf.v2;
+
+import com.sap.cloud.lm.sl.cf.core.cf.detect.mapping.ApplicationMetadataFieldExtractor;
+import com.sap.cloud.lm.sl.cf.core.cf.detect.mapping.MetadataFieldExtractor;
+import com.sap.cloud.lm.sl.cf.core.model.DeployedMtaModule;
+import com.sap.cloud.lm.sl.cf.core.model.DeployedMtaResource;
+import com.sap.cloud.lm.sl.cf.core.model.SupportedParameters;
+import com.sap.cloud.lm.sl.cf.core.util.NameUtil;
+import com.sap.cloud.lm.sl.common.util.JsonUtil;
+import com.sap.cloud.lm.sl.mta.model.DeploymentDescriptor;
+import com.sap.cloud.lm.sl.mta.model.Module;
+import com.sap.cloud.lm.sl.mta.model.ProvidedDependency;
+import com.sap.cloud.lm.sl.mta.model.Resource;
+import org.cloudfoundry.client.v3.Metadata;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class ApplicationMetadataBuilder {
```

Comment:
Maybe I am forgetting but why do we need this class?

Reply:
To extract the metadata building from application cloud model builder.